### PR TITLE
Adding a shell script to run all the workloads.

### PR DESCRIPTION
**Pull request by psudheer21**
Reviewers:
Neha, Karthik

### CODE REVIEW

File: run_sysbench.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pgip=127.0.0.1
```
Comment (ndeodhar) [inline run_sysbench.sh:1]: Can we make this configurable? Because this is something a lot of users may need to change.

File: run_sysbench.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+user=yugabyte
+db=yugabyte
+run_threads=64
+table_size=1000000
+time=120
+prefx='YB_1Mil_'
+
+load_workload() {
+  echo "LOADING"
+  time sysbench oltp_insert --tables=1 --table-size=$table_size --range_key_partitioning=true --serial_cache_size=1000 --db-driver=pgsql --pgsql-host=$pgip --pgsql-port=$port --pgsql-user=$user --pgsql-db=$db --threads=1 prepare > $prefx_load.txt
```
Comment (ndeodhar) [inline run_sysbench.sh:12]: Let's not output to a file but print to stdout instead.
It's not going to be obvious to users that we are writing to a file.

Comment (psudheer21) [reply run_sysbench.sh:12]: Done.

File: run_sysbench.sh (hunk 1 above)
Comment (psudheer21) [reply run_sysbench.sh:1]: Done.

**Pull request merged**